tests/solve-effort-flag.test.sh

```
@@ -0,0 +1,355 @@
+fi
+
+echo
+echo "== R2: invalid --effort=<level> values rejected loudly =="
+
+# R2a: typo like --effort=hgh aborts with non-zero exit and helpful stderr
+SANDBOX="$(mktemp -d)"
+mkdir -p "$SANDBOX/.claude"
+: > "$SANDBOX/.claude/uberdev.local.md"
+STDERR_FILE="$(mktemp)"
+(
+  cd "$SANDBOX"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)